### PR TITLE
Add description how install php-http-client manually

### PR DESCRIPTION
**Pull request by janczer**
According to #20. I added to `README.md` who install php-http-client without composer.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=25) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=25) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/php-http-client?pullRequest=25) it.</sub>

File: README.md [hunk 1]
```diff
@@ -38,16 +38,57 @@ Then from the command line:
...
+
+```
+$ cd /path/to/your/app
+$ mkdir lib
+$ cd lib
+$ git clone https://github.com/sendgrid/php-http-client.git
+$ git clone https://github.com/sendgrid/sendgrid-php.git
+```
+
+In the next step you shoud create `loader.php`:
```
Comment (mbernier) [inline README.md]: Please change "shoud" to "should"

Comment (janczer) [reply README.md]: Done

**Comment by thinkingserious** [comment]
Hello @janczer,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by janczer** [comment]
Hello @thinkingserious 
Thank you, I already filled out form for swag :)

**Pull request merged**